tests/network/hotplug_bridge.go

```diff
@@ -382,18 +382,22 @@ func removeInterface(vm *v1.VirtualMachine, name string) error {
 func verifyBridgeDynamicInterfaceChange(vmi *v1.VirtualMachineInstance, plugMethod hotplugMethod) *v1.VirtualMachineInstance {
 	virtClient := kubevirt.Client()
 	if plugMethod == migrationBased {
+		postOperationTimestamp := time.Now()
 		By("Waiting for MigrationRequired condition to appear")
 		Eventually(matcher.ThisVMI(vmi), 1*time.Minute, 2*time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceMigrationRequired))
 
 		By("Ensuring live-migration started")
 		var migration *v1.VirtualMachineInstanceMigration
 
 		Eventually(func() *v1.VirtualMachineInstanceMigration {
-			migrations, err := virtClient.VirtualMachineInstanceMigration(vmi.Namespace).List(context.Background(), metav1.ListOptions{})
+			migrations, err := virtClient.VirtualMachineInstanceMigration(vmi.Namespace).List(context.Background(), metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("%s=%s", v1.MigrationSelectorLabel, vmi.Name),
+			})
 			Expect(err).ToNot(HaveOccurred())
 
 			for _, mig := range migrations.Items {
-				if mig.Spec.VMIName == vmi.Name {
+				if mig.Spec.VMIName == vmi.Name &&
+					mig.CreationTimestamp.After(postOperationTimestamp) {
```

Comment:
isn't it theoretically raceful ?
the timestamp is sampled when we wait for migration to appear
but maybe it was created already by that time (after all the test can get starved versus the cluster running async)